### PR TITLE
fix: update ballot-encoder for new yes/no format

### PR DESCRIPTION
**Pull request by eventualbuddha**
I had to do this in order to generate updated fixtures for module-scan, but there's no urgency to review.

### CODE REVIEW

File: src/components/YesNoContest.tsx [hunk 1]
```diff
@@ -252,10 +257,7 @@ export default class YesNoContest extends React.Component<Props> {
   }
 
   public closeOvervoteAlert = () => {
-    // Delay to avoid passing tap to next screen
-    window.setTimeout(() => {
-      this.setState({ overvoteSelection: initialState.overvoteSelection })
-    }, 200)
+    this.setState({ overvoteSelection: initialState.overvoteSelection })
```
Comment (benadida) [inline src/components/YesNoContest.tsx:260]: I wouldn't want to remove the timeout right now unless we've done extensive testing. I know it *shouldn't* break, but this is super tricky.

Comment (eventualbuddha) [reply src/components/YesNoContest.tsx:260]: Fair enough, though we did remove the other ones:
- e302f200b40ce1fa1977eb45ee939a1fdb8151ad
- 841cebe559c4bb80c9345377a76453ed405add44

File: src/utils/votes.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { YesNoVote } from '@votingworks/ballot-encoder'
+import { YesNo } from '../config/types'
```
Comment (beausmith) [inline src/utils/votes.ts:2]: To make this even more clear…
```suggestion
import { YesOrNo } from '../config/types'
```

**Comment by beausmith** [comment]
What's status of this PR?

File: src/utils/votes.ts (hunk 2 above)
Comment (eventualbuddha) [reply src/utils/votes.ts]: Fixed.

**Review by beausmith — APPROVED** [review]

**Pull request merged**